data/beaconrestapi/src/test/java/tech/pegasys/artemis/beaconrestapi/beaconhandlers/GenesisTimeHandlerTest.java

```diff
@@ -13,30 +13,37 @@
 
 package tech.pegasys.artemis.beaconrestapi.beaconhandlers;
 
-import static org.assertj.core.api.Assertions.assertThat;
+import static javax.servlet.http.HttpServletResponse.SC_INTERNAL_SERVER_ERROR;
 
 import com.google.common.eventbus.EventBus;
 import com.google.common.primitives.UnsignedLong;
+import io.javalin.http.Context;
 import org.junit.jupiter.api.Test;
 import org.mockito.Mockito;
-import tech.pegasys.artemis.beaconrestapi.handlerinterfaces.BeaconRestApiHandler.RequestParams;
+import tech.pegasys.artemis.provider.JsonProvider;
 import tech.pegasys.artemis.storage.ChainStorageClient;
 
-class GenesisTimeHandlerTest {
-  private final RequestParams requestParams = Mockito.mock(RequestParams.class);
+public class GenesisTimeHandlerTest {
+  private Context mockContext = Mockito.mock(Context.class);
+  private final UnsignedLong genesisTime = UnsignedLong.valueOf(51234);
+
   private final ChainStorageClient storageClient =
       ChainStorageClient.memoryOnlyClient(new EventBus());
-  private final GenesisTimeHandler handler = new GenesisTimeHandler(storageClient);
 
   @Test
-  public void shouldReturnEmptyObjectWhenGenesisTimeIsNotSet() {
-    assertThat(handler.handleRequest(requestParams)).isEqualTo(null);
+  public void shouldRaiseInternalErrorWhenGenesisTimeIsNotSet() throws Exception {
+    GenesisTimeHandler subject = new GenesisTimeHandler(null);
```

Comment:
nit: We rarely use names like `subject` for test subjects in the codebase - typically this would just be `handler`.